content/en/blog/_posts/2022-11-11-retroactive-default-storage-class.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: blog
+title: "Retroactive default StorageClass"
+date: 2022-11-11
+slug: retroactive-default-storage-class
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+In Kubernetes 1.25 a new Alpha feature was introduced to improve the way default StorageClass is assigned to a PersistentVolumeClaim (PVC). It is no longer required to create a default StorageClass first and PVC second in order to assign the class and any PVCs without StorageClass can be updated later. The feature will be Beta in Kubernetes 1.26.
+
+See [Official Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retroactive-default-storageclass-assignment) for more details.
+
+## Why did StorageClass assignment need improvements
+
+When talking about PVCs and default StorageClasses you might think "It already works, when I create PVC it gets the default class just fine!" and that is correct because there already is a feature that causes admission controller to assign a class to **new** PVCs which is documented [here](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass).
```

Comment:
```suggestion
When considering PVCs and default StorageClasses you might think "It already works, when I create PVC it gets the default class just fine!" and that is correct because there already is an [admission controller](/docs/reference/access-authn-authz/admission-controllers/#defaultstorageclass)that assigns a StorageClass to **new** PVCs.
```